src/clients/travis-client.js

```diff
@@ -5,8 +5,22 @@ import { getActiveLogger } from '../utils/winston';
 const log = getActiveLogger();
 const tyrAgent = 'Travis/1.0';
 const travisApiUrl = 'https://api.travis-ci.org';
+const travisApiUrlPrivate = 'https://api.travis-ci.com';
```

Comment:
Move these inside the `getTravisApiUrl()` function to hide them from people using them incorrectly.